### PR TITLE
Configurable reverse synctex keybinding

### LINKED ISSUE #1596: Double click on PDF to jump to code
**Is your feature request related to a problem? Please describe.**

Coming from Overleaf, I often try to double click on the PDF to jump to the code. However, I need to use CMD+click. 

**Describe the solution you'd like**

Add double click as a trigger for synctex. 

**Describe alternatives you've considered**

It could be configurable. 

**Additional context**

This came up in https://github.com/James-Yu/LaTeX-Workshop/issues/662.

### PR DESCRIPTION
**Pull request by trevorgunn**
This has come up in several issues now:

#1596 
#1464 
#662
#106

With double clicking being the default on Overleaf and other online editors, I think it is worth having an option to have double clicking as a keybind.

Closes #1596

### CODE REVIEW

**Comment by tamuratak** [comment]
I do not like this approach. We should avoid that each new key binding requires a new `type` to be defined in the protocol. 

I prefer that, through a new type `keybinding`, key bindings configured by users are send to `latexworkshop.js`.

**Comment by trevorgunn** [comment]
@tamuratak Thank you for your comment. Please let me know if the latest commit resolves your concerns.

File: src/components/viewer.ts [hunk 1]
```diff
@@ -230,6 +230,7 @@ export class Viewer {
                             spreadMode: configuration.get('view.pdf.spreadMode'),
                             hand: configuration.get('view.pdf.hand'),
                             invert: configuration.get('view.pdf.invert'),
+                            keybinding: configuration.get('view.pdf.internal.synctex.keybinding')
```
Comment (tamuratak) [inline src/components/viewer.ts]: To make other keybindings configurable, please do not set the value of the keybinding of synctex directly.

I prefer:
```typescript
 keybinding: { synctex: configuration.get('view.pdf.internal.synctex.keybinding') }
```

**Comment by tamuratak** [comment]
It seems reasonable.

**Comment by domoritz** [comment]
Do you think you could support both using an array instead of a single value?

**Comment by James-Yu** [comment]
It seems that reverse synctex fails after the viewer is refreshed by a successful compilation. Any idea? @trevorgunn

**Pull request merged**